src/rand.rs

```diff
@@ -125,6 +125,10 @@ impl crate::sealed::Sealed for SystemRandom {}
 // system's) CSPRNG. Avoid using it on targets where it uses the `rdrand`
 // implementation.
 #[cfg(any(
+    // NOTE `getrandom`'s (v0.2.10) docs state that a custom implementation will
+    // NOT override the implementation of supported targets, like the ones
+    // listed below.
+    feature = "less-safe-getrandom-custom",
```

Comment:
I (we?) don't want people to this new "less-safe" feature as a shortcut to avoiding this allowlist, so this should be `all(feature = "less-safe-....", target_os = "none")`.